Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -957,47 +983,14 @@ script:
     - contextPath: URL.Category
       description: The URL category.
       type: String
-  - arguments:
-    - default: true
-      description: URL to check.
-      isArray: true
-      name: url
-      required: true
-    description: Gets a URL category from URL filtering.
```

Comment:
why was this removed?

Reply:
it wasn't removed, just changed its location to align with the new commands since this is a reputation command that will not be deprecated. see here:

![image](https://user-images.githubusercontent.com/53861351/154255020-1f6ea682-347f-420b-b58c-f2e8bc484639.png)